docs/index.rst

```diff
@@ -101,7 +101,6 @@ small example that shows adding a custom span to a Flask application::
         # Or just trace part of a function with the `trace`
         # context manager.
         with tracer.trace("thumbnails.save") as span:
-            span.set_meta("thumbnails.sizes", str(sizes))
```

Comment:
This method is implemented and works well.
https://github.com/DataDog/dd-trace-py/blob/6909a97184e00fc8e110a309230882a58d072dcb/ddtrace/span.py#L148